website/docs/docs/cloud/account-settings.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Account settings in dbt Cloud"
+sidebar_label: "Account settings" 
+description: "Learn how to enable account settings for your dbt Cloud users."
+---
+
+The following sections describe the different **Account settings** available from your dbt Cloud account in the sidebar (lower left-hand side). 
```

Comment:
It might be beneficial to add this since there are a few options there
```suggestion
The following sections describe the different **Account settings** available from your dbt Cloud account in the sidebar (under your account name on the lower left-hand side). 
```